splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/UnionOperationIT.java

```diff
@@ -451,6 +451,17 @@ public void testUnionUsingAliasInWhereClause() throws Exception {
         }
     }
 
+    @Test
```

Comment:
Does it make sense to make this test a micro-benchmark?

Reply:
@hatyo Perhaps, but it's not measuring for a specific level of performance, just a hang condition.  It either runs quickly, or hangs the ITs indefinitely which could flag a regression.  If ITs are executed more often than performance micro-benchmarks we may catch hang problems quicker.